crates/bevy_app/src/app.rs

```diff
@@ -814,6 +814,15 @@ impl App {
             .collect()
     }
 
+    /// This function is deprecated. Please use `add_plugin_group` instead.
+    #[deprecated(
+        since = "0.9.1",
```

Comment:
```suggestion
        since = "0.10.0",
```
It will be deprecated starting next release